modules/packages/LinearAlgebra.chpl

```diff
@@ -232,27 +232,28 @@ of the matrix.
 */
 class ExpmPadeHelper {
   type matType;
-  var A : matType;
-  var A2 : matType;
-  var A4 : matType;
-  var A6 : matType;
-  var A8 : matType;
-  var A10 : matType;
-  var exactD4 : real;
-  var exactD6 : real;
-  var exactD8 : real;
-  var exactD10 : real;
-  var approxD4 : real;
-  var approxD6 : real;
-  var approxD8 : real;
-  var approxD10 : real;
-  var ident : matType;
-  var useExactOneNorm : bool;
+  var A: matType;
```

Comment:
Could you add a comment either explaining how all this works or a link to something that does that?

Reply:
I see the comment in the user-facing doc, but I think something here could be useful for the reader, as well.